gradle/deploy.sh

```diff
@@ -15,12 +15,10 @@ elif [ $CIRCLE_TAG ]; then
   openssl aes-256-cbc -d -in gradle.properties.enc -out gradle.properties -k "$KEY" -md sha256
   case "$CIRCLE_TAG" in
   *-rc\.*)
-    ./gradlew -Prelease.disableGitChecks=true -Prelease.useLastTag=true candidate bintrayUpload $SWITCHES -x release -x bintrayPublish
-    ./gradlew -Prelease.disableGitChecks=true -Prelease.useLastTag=true :micrometer-core:bintrayPublish $SWITCHES -x bintrayUpload
+    ./gradlew -Prelease.disableGitChecks=true -Prelease.useLastTag=true candidate publishAllPublicationsToMilestoneRepository $SWITCHES -x release -x artifactoryPublish -x bintrayUpload -x bintrayPublish
```

Comment:
publishAll is probably fine, but I wonder if it wouldn't be better to be more specific just in case there is more than one publication registered. Would we want to publish all of them? Can't we publish the nebula publication specifically since that's the one we expect to be there?